pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[build-system]
+requires = ["hatchling>=1.25"]
+build-backend = "hatchling.build"
+
+[project]
+name = "model_signing"
+dynamic = ["version"]
+description = "A tool for signing and verifying ML models"
+readme = "README.model_signing.md"  # TODO: Merge READMEs and update
+license = { file = "LICENSE" }
+authors = [
+  { name = "Sigstore Authors", email = "sigstore-dev@googlegroups.com" }
+]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Developers",
+  "Topic :: Security",
+  "Topic :: Security :: Cryptography",
+  "Topic :: Scientific/Engineering :: Artificial Intelligence",
+  "Typing :: Typed",
+]
+dependencies = [
+  "cryptography",  # TODO: make pki a feature
+  "in-toto-attestation",
+  "sigstore",
+  "typing_extensions",
+]
```

Comment:
this isn't a match from the old `requirements.in`?

Reply:
It isn't. These are just the dependencies needed to make sure the wheel can be installed and used.

`requirements.in` should be used for development scenarios. We were splitting them into requirements for the CLI POCs and for requirements for testing, but we don't need that